### PR TITLE
Use static jquery reference in soil base template

### PR DESCRIPTION
**Pull request by gherceg**
## Summary

[JIRA Ticket](https://dimagi-dev.atlassian.net/browse/SAAS-11751)

Because we pulled jquery 1.9.1 directly from a cdn, we missed out on security vulnerability warnings from dependabot. This will allow us to better manage any v1 reference to jquery.

Due to a cve in 1.9.1, we are upgrading to 1.12.4 (the latest version of jquery v1 at the moment).
## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### QA Plan

Will request QA.
### Safety story

Locally testing now. Will test on staging as well.
### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

**Comment by orangejenny** [comment]
Have you looked into just switching this to the same jQuery 3 the rest of HQ is using? It's unlikely this page needs to use 1, it's probably an oversight that it hasn't been upgraded.

Looking a bit more closely, the only descendant template I see is `soil/file_download.html`, which is using `javascript_libraries` which pulls in jQuery 3, so I suspect this page is already loading both 1 and 3 and you could just remove 1. The only js used on file_download.html is [file_download.js](https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/soil/static/soil/js/file_download.js) which is a pretty small amount of js that would be fine on jQuery 3.

**Comment by gherceg** [comment]
@orangejenny definitely worth a shot! Thanks.

File: corehq/ex-submodules/soil/templates/soil/base.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
```
Comment (biyeun) [inline corehq/ex-submodules/soil/templates/soil/base.html:3]: ```{% load hq_shared_tags %}``` is missing from the top here. that's where `static` comes from

**Comment by orangejenny** [comment]
I think your smoke testing sounds sufficient, don't see a need for more formal QA.

**Review by biyeun — APPROVED** [review]

**Pull request merged**